src/north/CsvToHttp/CsvToHttp.class.spec.js

```diff
@@ -0,0 +1,140 @@
+const path = require('path')
+const CsvToHttp = require('./CsvToHttp.class')
+const ApiHandler = require('../ApiHandler.class')
+const config = require('../../../tests/testConfig').default
+const Logger = require('../../engine/Logger.class')
+const EncryptionService = require('../../services/EncryptionService.class')
+
+// Mock logger
+jest.mock('../../engine/Logger.class')
+Logger.getDefaultLogger = () => new Logger()
+
+// Mock EncryptionService
+EncryptionService.getInstance = () => ({ decryptText: (password) => password })
+
+// Mock engine
+const engine = jest.genMockFromModule('../../engine/Engine.class')
+engine.configService = { getConfig: () => ({ engineConfig: config.engine }) }
+engine.sendRequest = jest.fn()
+engine.decryptPassword = (password) => password
+
+// Define the CsvToHttp North
+const CsvToHttpConfig = config.north.applications[6]
+const CsvToHttpNorth = new CsvToHttp(CsvToHttpConfig, engine)
+
+beforeEach(() => {
+  jest.resetAllMocks()
+  jest.clearAllMocks()
+  CsvToHttpNorth.httpBody = []
+})
+
+describe('CsvToHttp', () => {
+  it('should be properly initialized', () => {
+    expect(CsvToHttpNorth.canHandleValues).toBeFalsy()
+    expect(CsvToHttpNorth.canHandleFiles).toBeTruthy()
+  })
+
+  it('should properly handle files (not csv one)', async () => {
+    const response = await CsvToHttpNorth.handleFile('../../../tests/csvToHttpTest')
+
+    expect(response).toEqual(ApiHandler.STATUS.LOGIC_ERROR)
+  })
+
+  it('should properly handle files (csv one)', async () => {
+    const response = await CsvToHttpNorth.handleFile(path.resolve('./tests/csvToHttpTest.csv'))
+
+    expect(response).toEqual(ApiHandler.STATUS.SUCCESS)
+  })
+
+  it('should properly convert values', async () => {
```

Comment:
This test should be adapted to the latest changes (it always return objects, with errors if a problem occurs)